cmd/nerdctl/image_encrypt_linux_test.go

```diff
@@ -97,15 +96,8 @@ func rmiAll(base *testutil.Base) {
 		}
 
 		base.T.Logf("Pruning all images (again?)")
-		is := client.ImageService()
-		imgs, err := is.List(ctx)
-		assert.NilError(base.T, err)
-		for _, img := range imgs {
-			base.T.Logf("Pruning image %+v", img)
-			if err := is.Delete(ctx, img.Name, images.SynchronousDelete()); err != nil {
-				base.T.Log(err)
-			}
-		}
+		imageIDs = base.Cmd("images", "--no-trunc", "-a", "-q").OutLines()
+		base.Cmd(append([]string{"rmi", "-f"}, imageIDs...)...).AssertOK()
 	}
```

Comment:
?

Reply:
It looks like that the recent `nerdctl rmi` [strictlly requires each container's image exists in the image service](https://github.com/containerd/nerdctl/blob/d39e9c8137837398395e8af62db0d879ee69b13b/pkg/cmd/image/remove.go#L49-L52) and it has a logic to avoid removing images of existing containers.
This part of code didn't seem to meet this requirement and it seems to result in CI failures that report [a container's corresponding image doesn't exist in the image store](https://github.com/containerd/nerdctl/actions/runs/4016209793/jobs/6898926792#step:5:9468).
So this part tries to fix it by using `nerdctl rmi`.